pipelined/bevy_sprite2/src/texture_atlas.rs

```diff
@@ -22,30 +22,19 @@ pub struct TextureAtlas {
 
 #[derive(Debug, Clone, TypeUuid, Reflect)]
 #[uuid = "7233c597-ccfa-411f-bd59-9af349432ada"]
-pub struct TextureAtlasSprite {
-    pub color: Color,
+pub struct TextureAtlasEntry {
     pub index: u32,
-    pub flip_x: bool,
-    pub flip_y: bool,
```

Comment:
I really like how this simplification resulted from including `Sprite2d`/`Sprite3d` in the sprite sheet bundles. Nice bit of code reuse.